pkg/controller/snapshot/cron.go

```diff
@@ -101,17 +101,6 @@ func (c *cronController) ScheduleBackup(
 	// Remove previous cron job if exist
 	if id, exists := c.cronEntryIDs.Pop(cronEntryName); exists {
 		c.cron.Remove(id.(cron.EntryID))
-	} else {
```

Comment:
This check fails due when operator restarts due to mutation webhook is not ready. So, we just avoid the check.